app/controllers/verify/step_controller.rb

```diff
@@ -8,5 +8,28 @@ class StepController < ApplicationController
     before_action :confirm_idv_attempts_allowed
 
     helper_method :step
+
+    private
+
+    def step_name
+      raise NotImplementedError, 'must implement step_name method'
+    end
```

Comment:
The purpose of this controller is to reduce duplication since several controllers use the methods defined here. That to me signals this should be a controller concern, not a superclass that other controllers inherit from.